ansible/workers.yml

```diff
@@ -149,7 +150,7 @@
     pip:
       name:
       - pytest
-    when: ansible_distribution == 'RedHat'
+      when: ansible_distribution == 'RedHat'
```

Comment:
this indent is misleading, the `when` applies to the whole task, not just the pip subtask.